index.html

```diff
@@ -507,22 +507,42 @@ <h3 id="bp-producers">Guidance for [=Producers=]</h3>
 <section id="guidance-for-consumers">
 <h3 id="bp-consumers">Guidance for [=Consumers=]</h3>
 
+<p>The purpose of collecting, serializing, and transmitting language and [=string direction=] metadata is so that [=consumers=] can use it to display and process string data correctly.</p>
+
 <div class="req" id="bp-consumer-lang-assumption">
-<p class="advisement">Consumers SHOULD consume language metadata provided by document formats or protocols whenever possible. When language metadata is not available, consumers MAY use heuristics to detect the language.</p>
+<p class="advisement">Consumers SHOULD employ any language metadata provided by document formats or protocols when processing or displaying the associated string value.</p>
 </div>
 
 <div class="req" id="bp-consumer-dir-assumption">
-<p class="advisement">Consumers SHOULD consume direction metadata provided by document formats or protocols whenever possible. When direction metadata is not available, consumers SHOULD use first-strong detection (possibly by using the value <code>auto</code> for the direction).</p>
+<p class="advisement">Consumers SHOULD employ any [=string direction=] metadata provided by document formats or protocols when processing or displaying the associated string value.</p>
 </div>
 
 <div class="req" id="bp-consumer-isolation">
-<p class="advisement">When inserting content into a document, and particularly when inserting that content into existing text, consumers SHOULD apply bidirectional isolation via markup or Unicode isolating characters. Such isolation SHOULD use the direction found in document format or protocol supplied metadata if possible.</p>
+<p class="advisement">When displaying a string or inserting a string into a document, consumers SHOULD apply bidirectional isolation to the string value using the [=string direction=] found in document format or protocol metadata when available or first-strong isolation when it is not.</p>
```

Comment:
This is a very long, meandering sentence, that packs in a lot of information, but in a way that is not very easy to read.  I suggest replacing this mustard with two mustards as follows:

````
<div class="req" id="bp-consumer-isolation">
<p class="advisement">When a string is displayed in or inserted into a document, consumers SHOULD isolate it directionally from the surrounding text.</p>
</div>`

<div class="req" id="bp-consumer-direction">
<p class="advisement">Consumers SHOULD apply direction to a string when it is inserted into a document. If [=string direction=] is provided by metadata associated with a string, consumers SHOULD use that. If string metadata is not available, first-strong heuristics should be used to assign a direction.</p>
</div>
````

Reply:
@r12a If you're going to go to the trouble of typing you suggestion out, consider using \`\`\`suggestion to mark up the suggested text. Like so:

```suggestion
<div class="req" id="bp-consumer-isolation">
<p class="advisement">When a string is displayed in or inserted into a document, consumers SHOULD isolate it directionally from the surrounding text.</p>
</div>`

<div class="req" id="bp-consumer-direction">
<p class="advisement">Consumers SHOULD apply direction to a string when it is inserted into a document. If [=string direction=] is provided by metadata associated with a string, consumers SHOULD use that. If string metadata is not available, first-strong heuristics SHOULD be used to assign a direction.</p>
</div>
```